nifi-nar-bundles/nifi-hive-bundle/nifi-hive3-processors/src/main/java/org/apache/hive/streaming/NiFiRecordSerDe.java

```diff
@@ -205,7 +212,9 @@ private Object extractCurrentField(Record record, RecordField field, TypeInfo fi
                         val = record.getAsString(fieldName);
                         break;
                     case BINARY:
-                        val = AvroTypeUtil.convertByteArray(record.getAsArray(fieldName)).array();
+                        Object[] array = record.getAsArray(fieldName);
+                        if (array == null) return null;
```

Comment:
I'm surprised this passes checkstyle. I thought we required the {} with the body to start on the next line... is preferred syntax anyway I think.